audio_service/lib/audio_service.dart

```diff
@@ -28,75 +28,111 @@ enum MediaButton {
 enum MediaAction {
   /// Stop playing audio.
   stop,
+  // 0
 
   /// Pause audio.
   pause,
+  // 1
 
   /// Play or resume audio.
   play,
+  // 2
 
   /// Rewind.
   rewind,
+  // 3
 
   /// Skip to the previous media item.
   skipToPrevious,
+  // 4
 
   /// Skip to the next media item.
   skipToNext,
+  // 5
 
   /// Fast forward.
   fastForward,
+  // 6
 
   /// Set a rating for the current media item.
   setRating,
+  // 7
 
   /// Seek within the current media item.
   seek,
+  // 8
 
   /// Toggle between playing and paused.
   playPause,
+  // 9
 
   /// Play a given media item by ID.
   playFromMediaId,
+  // 10
 
   /// Play media from a search.
   playFromSearch,
+  // 11
 
   /// Skip to a queue item.
   skipToQueueItem,
+  // 12
 
   /// Play media from a URI.
   playFromUri,
+  // 13
 
   /// Prepare media for playback.
   prepare,
+  // 14
 
   /// Prepare media for playback by ID.
   prepareFromMediaId,
+  // 15
 
   /// Prepare media for playback from a search.
   prepareFromSearch,
+  // 16
 
   /// Prepare media for playback from a URI.
   prepareFromUri,
+  // 17
 
   /// Set the repeat mode.
   setRepeatMode,
+  // 18
 
-  /// Unused.
-  unused_1,
+  /// Set captioning enabled.
+  setCaptioningEnabled,
+  // 19
 
-  /// Unused.
-  unused_2,
+  // On Android corresponds to the deprecated `SET_SHUFFLE_MODE_ENABLED`
+  _unused_2,
+  // 20
 
   /// Set the shuffle mode.
   setShuffleMode,
+  // 21
+
+  /// Set playback speed.
+  setSpeed,
+  // 22
 
   /// Seek backwards continuously.
   seekBackward,
+  // 23
 
   /// Seek forwards continuously.
   seekForward,
+  // 24
+
+  /// Unused.
+  @Deprecated("Doesn't do anything. Will be removed in the next release")
```

Comment:
This can't be deprecated until https://github.com/dart-lang/language/issues/158 is implemented. However, in the meantime, we can make these enum values private as per https://github.com/dart-lang/language/issues/1151

Reply:
Well I suppose it could be deprecated, but not removed.

Reply:
I will keep one unused action that is necessary, the other should be removed

Reply:
Wasn't aware of this thing with private enum members and switch

Reply:
Ah, that's problematic. Maybe we can just remove them without deprecation, since that's impossible currently?